controllers/expenses.controller.js

```diff
@@ -0,0 +1,10 @@
+const { expenses } = require("../models");
+
+exports.get = async (req, res) => {
+  try {
+    const all = await expenses.findAll();
+    res.json(all);
+  } catch (err) {
+    console.error(err.message);
```

Comment:
error must be handlable on front so send it back in json format and also you must change status code, because default is 200 and in case of error there got to be another